README.md

```diff
@@ -7,48 +7,45 @@
 # Imitation Learning Baseline Implementations
 
 This project aims to provide clean implementations of imitation learning algorithms.
-Currently we have implementations of [AIRL](https://arxiv.org/abs/1710.11248) and 
-[GAIL](https://arxiv.org/abs/1606.03476), and intend to add more in the future.
+Currently we have implementations of Behavioral Cloning, [DAgger](https://arxiv.org/pdf/1011.0686.pdf) (with synthetic examples), [Adversarial Inverse Reinforcement Learning](https://arxiv.org/abs/1710.11248), and [Generative Adversarial Imitation Learning](https://arxiv.org/abs/1606.03476).
 
 ### To install:
 ```
-conda create -n imitation python=3.7
-conda activate imitation
-pip install -e '.[dev]'  # install `imitation` in developer mode
+git clone http://github.com/HumanCompatibleAI/imitation
+cd imitation
+pip install -e .
 ```
 
 ### Optional Mujoco Dependency:
 
-Follow instructions to install [mujoco_py v1.5 here](https://github.com/openai/mujoco-py/tree/498b451a03fb61e5bdfcb6956d8d7c881b1098b5#install-mujoco).
+Follow instructions to install [mujoco\_py v1.5 here](https://github.com/openai/mujoco-py/tree/498b451a03fb61e5bdfcb6956d8d7c881b1098b5#install-mujoco).
 
-### To run:
-```
-# Train PPO agent on cartpole and collect expert demonstrations
-python -m imitation.scripts.expert_demos with cartpole
-# Train AIRL on from demonstrations
-python -m imitation.scripts.train_adversarial with cartpole airl
+
+## Sacred CLI Quickstart:
```

Comment:
Most people won't know what Sacred is. How about just "CLI Quickstart" as heading, then we can have an intro sentence saying:

```
We provide several CLI scripts as a front-end to the algorithms implemented in `imitation`. These use [Sacred](https://github.com/idsia/sacred) for configuration and replicability.
```